professor2.spec

```
@@ -1,15 +1,16 @@
-### RPM external professor2 2.3.3
+### RPM external professor2 2.4.2
 ## INITENV +PATH PYTHON3PATH %i/lib/python%{cms_python3_major_minor_version}/site-packages
+## INITENV +PATH PROF_VERSION %{realversion}
 ## INCLUDE cpp-standard
 
-Source: http://www.hepforge.org/archive/professor/Professor-%{realversion}.tar.gz
-Requires: py3-matplotlib root yoda eigen
+Source: http://www.hepforge.org/archive/professor/professor-professor-%{realversion}.tar.gz
```

```suggestion
Source: git+https://gitlab.com/hepcedar/professor.git?obj=main/professor-%{realversion}&export=professor-%{realversion}&output=/professor-%{realversion}.tgz
```

Hi Sitian, I'd suggest to move to git, as the new tarball name looks more like an accident

---

Yeah one of the confusing part, let me try